bindings/nodejs/src/lib.rs

```diff
@@ -582,6 +615,56 @@ impl Operator {
     }
 }
 
+macro_rules! ww {
```

Comment:
Hi, I'm personally not a fan of `macro_rules`. How about repeating those code? They won't be changed often I guess.

Reply:
ok, removed